Making dependencies more flexible, adding version, disabling SCM (not working with EasyBuild)
--- setup_orig.py	2023-07-24 15:21:02.016292082 +0200
+++ setup.py	2023-07-24 15:45:57.822276166 +0200
@@ -1,10 +1,10 @@
 import setuptools
 from setuptools import setup
 
-install_deps = ['numpy>=1.22.4', 'scipy', 'natsort',
+install_deps = ['numpy>=1.22.3', 'scipy', 'natsort',
                 'tifffile', 'tqdm', 'numba', 
                 'torch>=1.6',
-                'opencv-python-headless', # headless not working with pyinstaller 
+                'opencv-contrib-python', # headless not working with pyinstaller 
                 'fastremap', 'imagecodecs'
                 ]
 
@@ -57,6 +57,7 @@
     
 setup(
     name="cellpose-omni",
+    version='0.9.1',
     license="BSD",
     author="Kevin Cutler",
     author_email="kevinjohncutler@outlook.com",
@@ -69,7 +70,7 @@
       'setuptools_scm',
     ],
     packages=setuptools.find_packages(),
-    use_scm_version=True,
+    use_scm_version=False,
     install_requires = install_deps,
     tests_require=[
       'pytest'
